'''What will the following code do and why? Don't run it until 
you have tried to answer.

x = 10

def my_function():
    x = x + 5
    print(x)

my_function()
'''

'''Answer:

It will raise an error because x is not defined within the fuction.
We would need to use global x if we wanted to use the x defined 
outside of the function.
'''